go/vt/vtctl/workflow/server.go

```diff
@@ -299,7 +299,8 @@ func (s *Server) GetCellsWithShardReadsSwitched(
 // keyspace.
 func (s *Server) GetCellsWithTableReadsSwitched(
 	ctx context.Context,
-	keyspace string,
+	sourceKeyspace string,
+	targetKeyspace string,
```

Comment:
I don't see where we're using targetKeyspace.

Reply:
I'm surprised that this is passing the linter though.

Reply:
The linter doesn't complain about unused function parameters. We have a lot of functions, for example, which take a `ctx` (because many of the functions around it have it as the first param) but don't use it.